src/gazebo_gps_plugin.cpp

```diff
@@ -85,7 +85,6 @@ void GpsPlugin::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
   updateConnection_ = event::Events::ConnectWorldUpdateBegin(
       boost::bind(&GpsPlugin::OnUpdate, this, _1));
 
-  last_gps_time_ = world_->GetSimTime();
```

Comment:
Was this intended?

Reply:
It's not. @arwagoner ^

Reply:
It was intended because this line is also here https://github.com/PX4/sitl_gazebo/blob/68f661506a6ffd653125a3e6327b462d80baa81f/src/gazebo_gps_plugin.cpp#L47

Reply:
> It was intended because this line is also here https://github.com/PX4/sitl_gazebo/blob/68f661506a6ffd653125a3e6327b462d80baa81f/src/gazebo_gps_plugin.cpp#L47

Yeah makes sense